utils/build/docker/nodejs/express4/integrations/api_gateway.js

```diff
@@ -0,0 +1,14 @@
+function initRoutes (app, tracer) {
+  console.log('Api Gateway routes initialized.')
+
+  app.get('/api-gateway/span-creation', (req, res) => {
```

Comment:
Could you document in `docs/weblog` what this endpoint is supposed to do ?

Reply:
done